CHANGELOG.md

```diff
@@ -20,6 +23,14 @@ file to the new `mwa_metafits_file` parameter.
 - Support for setting the astrometry library for various object methods including `set_lsts_from_time_array`, file read methods and others.
 - Properly round-trip the telescope frame through UVH5, UVFITS and MS files.
 
+### Changed
+- Increased the tolerance to 15 mas for a warning about values in `lst_array` not conforming
+to exceptations for `UVData`, `UVCal`, and `UVFlag` (was 1 mas) inside of `check`, additionally
+adding a keyword to enable the tolerance value to be specified.
+- Changed the behavior of checking of telescope location to look at the combination of
+`antenna_positions` and `telescope_location` together for `UVData`, `UVCal`, and `UVFlag`.
+Additionally, failing this check results in a warning (was an error).
```

Comment:
These should go under the `Unreleased` section.

Reply:
Fixed!